exercises/armstrong-numbers/.meta/generator/armstrong_numbers_case.rb

```diff
@@ -0,0 +1,36 @@
+require 'generator/exercise_case'
+
+class ArmstrongNumbersCase < Generator::ExerciseCase
+
+  def workload
+    case expected
+    when true  then true_workload
+    when false then false_workload
+    else raise "unexpected 'expected' value encountered"
+    end
+  end
```

Comment:
@guygastineau You can simplify this a bit if you use the `assert_or_refute` helper defined [here](https://github.com/exercism/ruby/blob/master/lib/generator/exercise_case/assertion.rb#L16).